src/styles/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
```

Comment:
Ensure that the variable `$gap-size` is defined in your variables file and follows the Code Style Rules for naming variables (should be kebab-case).